nilearn/image/image.py

```diff
@@ -502,6 +502,58 @@ def swap_img_hemispheres(img):
 
     # create swapped nifti object
     out_img = nibabel.Nifti1Image(img.get_data()[::-1], img.get_affine(),
-        header=img.get_header())
+                                  header=img.get_header())
 
     return out_img
+
+
+def _index_img(imgs, index):
+    """Helper function for index_img and iter_img
+    """
+    return nibabel.Nifti1Image(imgs.get_data()[..., index],
```

Comment:
:+1: 

(clueless users don't know the difference between slicing and fancy indexing, and probably won't even think about assigning values to a nifti slice. Experienced users would probably be annoyed if the behaviour of slicing was changed.)